scripts/setup/bootstrap.sh

```diff
@@ -56,6 +56,29 @@ _install_additional_pip_requirements() {
     unset _SETUP_PLATFORM
 }
 
+_submodules_need_updating() {
+  # Validates if a set o submodules that should always be checked out are up to date
+
+  # Pigweed will be up to date on an initial setup, however it may change over time
+  # the rest are a small subset of things that are always checked out (have no platform attachment)
```

Comment:
Uppercase "the", please, and probably periods at the ends of sentences in a multi-sentence comment?